examples/nemo/scripts/kernels_trans.py

```diff
@@ -119,7 +119,7 @@
                       "psi_h", "psi_m", "psi_m_coare",
                       "psi_h_coare", "psi_m_ecmwf", "psi_h_ecmwf",
                       "q_sat", "rho_air",
-                      "Ri_bulk", "visc_air", "sbc_dcy", "glob_sum",
```

Comment:
Is `Ri_bulk` not a function?

Reply:
Please update the copyright date at the top of this file and add yourself as author.

Reply:
> Is `Ri_bulk` not a function?

It is, but since it is capitalised and the script checks if `ref.name.lower() in NEMO_FUNCTIONS`, its presence is inconsequential.